src/gui/qgsmapcanvas.cpp

```diff
@@ -2301,7 +2300,7 @@ void QgsMapCanvas::stopPreviewJobs()
 void QgsMapCanvas::schedulePreviewJob( int number )
 {
   mPreviewTimer.setSingleShot( true );
-  mPreviewTimer.setInterval( 250 );
+  mPreviewTimer.setInterval( PREVIEW_JOB_DELAY_MS );
```

Comment:
How about making this configurable with QgsSettings? I think this was an arbitrary default and I'd love to have some power users test different settings to see what's reasonable.